src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTask.java

```diff
@@ -51,35 +56,24 @@
 
     private static final Logger log = LoggerFactory.getLogger(BlobProcessorTask.class);
 
-    @Value("${storage.blob_lease_timeout}")
-    private Integer blobLeaseTimeout;
-
-    @Value("${storage.blob_processing_delay_in_minutes}")
-    protected int blobProcessingDelayInMinutes = 0;
+    // TODO check if this is used somewhere or if I removed it accidentally
+    private final Integer blobLeaseTimeout;
+    private final int blobProcessingDelayInMinutes;
 
     @Autowired
     public BlobProcessorTask(
-        CloudBlobClient cloudBlobClient,
-        DocumentProcessor documentProcessor,
-        EnvelopeProcessor envelopeProcessor,
-        ErrorHandlingWrapper errorWrapper
-    ) {
-        super(cloudBlobClient, documentProcessor, envelopeProcessor, errorWrapper);
-    }
-
-    // NOTE: this is needed for testing as children of this class are instantiated
-    // using "new" in tests despite being spring beans (sigh!)
-    public BlobProcessorTask(
-        CloudBlobClient cloudBlobClient,
+        AzureStorageHelper azureStorageHelper,
```

Comment:
Wow quite a few parameters. 
Maybe a parameter object or properties object would be better ?